.github/workflows/NuGet Solution Master Workflow.yml

```diff
@@ -192,7 +192,9 @@ jobs:
 
     # Signing cannot be done from linux environment (https://github.com/dotnet/runtime/issues/48794)
   sign:
-      # The type of runner that the job will run on
+    # Don't run the signing when dependabot branch/pull request
+    if: ${{ github.secret_source != 'Dependabot' }}
```

Comment:
This is a bit strange, to use the secret source. But if it works, it works I suppose.
I did search around if there were other ways being used or recommended and I found something documented in dependabot itself:

    if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#responding-to-events

Reply:
Changed it so it looks at the actor. I didn't include the preview dependabot as we don't use that.